docs/supported_profilers/terminology.md

```diff
@@ -0,0 +1,92 @@
+# Terminology
+
+!!! info
+
+    Even though we have very specific concepts in mind when talking about the
+    different tools and their results, we don't (yet) apply this terminology
+    consistently everywhere. We will do so as time permits. In the meantime,
+    please study the concepts presented herein carefully and form your own
+    judgement how they apply elsewhere in the documentation.
+
+Our thinking about terminology is strongly influenced by the categorisation proposed by Sun _et al._ (2021)[^1].
+To summarise: Broadly, we call all of these tools _metagenomic profilers_. They can be
+more finely grouped by considering their type of reference database and thus the kind of relative abundance profile
+produced. Furthermore, the underlying method for sequencing read classification impacts
+the [computational complexity](https://en.wikipedia.org/wiki/Computational_complexity) (time and database size/memory)
+dramatically.
+
+[^1]: Sun, Z., Huang, S., Zhang, M., Zhu, Q., Haiminen, N., Carrieri, A. P., Vázquez-Baeza, Y., Parida, L., Kim, H.-C., Knight, R., & Liu, Y.-Y. (2021). Challenges in benchmarking metagenomic profilers. _Nature Methods_. <https://doi.org/10.1038/s41592-021-01141-3>
+
+## Criteria
+
+### Type of Abundance/Profiler
+
+Our main criteria for categorizing a metagenomic profiler are summarised in the list below,
+as well as shown in the schema.
+
+-   Reference database:
+    -   **DNA-to-DNA**, aka a metagenome library
+    -   **DNA-to-AA**, aka a genomic library of protein coding sequences
+    -   **DNA-to-marker**, aka marker gene, aka gene families selection
+-   Result:
+    -   relative **sequence abundance**
+    -   relative **protein sequence abundance**
+    -   relative **taxonomic abundance**
+
+```mermaid
+graph TB
+
+    P(Metagenomic Profiler)
+    D(DNA-to-DNA)
+    A(DNA-to-AA)
+    M(DNA-to-marker)
+    S(Sequence Abundance)
+    SAA(Protein Sequence Abundance)
+    T(Taxonomic Abundance)
+    TP(Taxonomic Profiler)
+    SP(Sequence Profiler aka Classifier)
+
+    P --> D
+    P --> A
+    P --> M
+    D --> S
+    A --> SAA
+    M --> T
+    S --> SP
+    SAA --> SP
+    T --> TP
+```
+
+### Differences in Abundance
+
+A key takeaway from Sun _et al._ (2021)[^1] is also that the different kinds of relative
+abundances cannot easily be interconverted by, for example, normalizing sequence abundance by genome size.
+Reasons for this include, in particular for metagenome-assembled genomes (MAGs), inaccurate
```

Comment:
```suggestion
Reasons for this include (in particular for metagenome-assembled genomes (MAGs)) inaccurate
```

Reply:
I prefer less parentheses 😬

Reply:
Ok maybe rephrase then - I found it hard to read initially thinking the mags were a part of the list itself not a emphasis